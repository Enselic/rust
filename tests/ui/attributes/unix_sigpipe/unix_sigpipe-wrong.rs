//@ revisions: with_feature without_feature

#![cfg_attr(with_feature, feature(unix_sigpipe))]

#[unix_sigpipe = "wrong"] //~ error: the only valid variant of the `unix_sigpipe` attribute is `#[unix_sigpipe = "sig_dfl"]
fn main() {}
