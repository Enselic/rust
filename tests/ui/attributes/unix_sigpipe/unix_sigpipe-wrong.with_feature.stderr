error: the only valid variant of the `unix_sigpipe` attribute is `#[unix_sigpipe = "sig_dfl"]`
  --> $DIR/unix_sigpipe-wrong.rs:5:1
   |
LL | #[unix_sigpipe = "wrong"]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 1 previous error

